### PR TITLE
Move TTL 74266 to 747266, make 74266 open-collector

### PR DESCRIPTION
**Pull request by btb**
The 74x266 IC has open-collector outputs, which means it isn't capable of driving its outputs high. This lets you connect its outputs directly together with other OC outputs, and needs a pull-up resistor to bring the level high.

This PR renames it to 74x7266, which is the non-open-collector equivalent.

We implement the actual 74x266 by making it output Value.UNKNOWN instead of Value.TRUE, and adding the schematic symbol for open-collector to the painted symbol.

There are many other TTL series chips with OC outputs that could be similarly implemented

### CODE REVIEW

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]
Please address linter complaints.

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74266.java [hunk 1]
```diff
@@ -52,10 +49,10 @@ public void paintInternal(InstancePainter painter, int x, int y, int height, boo
   @Override
   public void propagateTtl(InstanceState state) {
     for (byte i = 2; i < 6; i += 3) {
-      state.setPort(i, (state.getPortValue(i - 1).xor(state.getPortValue(i - 2)).not()), 1);
+      state.setPort(i, (state.getPortValue(i - 1).xor(state.getPortValue(i - 2)).not() == Value.TRUE)?Value.UNKNOWN:Value.FALSE, 1);
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/std/ttl/Ttl74266.java:52]: Please add spaces around operators `?` and `:`.

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74266.java [hunk 2]
```diff
@@ -52,10 +49,10 @@ public void paintInternal(InstancePainter painter, int x, int y, int height, boo
...
     }
     for (byte i = 6; i < 12; i += 3) {
-      state.setPort(i, (state.getPortValue(i + 1).xor(state.getPortValue(i + 2)).not()), 1);
+      state.setPort(i, (state.getPortValue(i + 1).xor(state.getPortValue(i + 2)).not() == Value.TRUE)?Value.UNKNOWN:Value.FALSE, 1);
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/std/ttl/Ttl74266.java:55]: Please add spaces around operators `?` and `:`.

**Review by maehne — DISMISSED** [review]
Looks mostly good to me except for small details

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74266.java (hunk 1 above)
Comment (btb) [reply src/main/java/com/cburch/logisim/std/ttl/Ttl74266.java]: done

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74266.java (hunk 2 above)
Comment (btb) [reply src/main/java/com/cburch/logisim/std/ttl/Ttl74266.java]: done

**Review by maehne — APPROVED** [review]
Looks good to me!

**Comment by maehne** [comment]
@MarcinOrlowski: What points still need to be addressed before merging this PR?

**Comment by BFH-ktt1** [comment]
@MarcinOrlowski : What is still blocking this PR?

**Pull request merged**